src/shared/helpers/__tests__/converter.test.ts

```diff
@@ -30,7 +30,7 @@ describe("Converter File", () => {
     const expected = {
       family: "ethereum",
       amount: new BigNumber(0),
-      recipient: eip55.encode(ethTx.to),
+      recipient: ethTx.to ? eip55.encode(ethTx.to) : "",
```

Comment:
Shouldn't we add a case where to is undefined instead of modifying this one ?

Reply:
agree, your comment makes totally sense

Reply:
Could you provide one test without the recipient an one with it, please?
Because here it will always be defined as it comes from a mock